sorachung/pet-journal
# Src medical

```text
#### Changes Made
1. Modified file `MedicalRepository` to fix/edit a bunch of fetch calls
2. Modified file `ViewMedical` to include `viewing medical dashboard` and functionality.
3. Modified file `IncidentsList and Incident` to include `viewing incidents` and functionality.
4. Modified file `VetVisitsList and VetVist` to include `viewing vet visits` and functionality.
5. Modified file `PetMedicalBio` to include `viewing pet medical bio` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout src-medical
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > start app
    > log in as sora
    > go to medical dashboard from side bar
    > go nuts
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
